prov/gni/src/gnix_ep.c

```diff
@@ -113,21 +113,31 @@ static inline ssize_t __ep_recv(struct fid_ep *ep, void *buf, size_t len,
 
 static inline ssize_t __ep_recvv(struct fid_ep *ep, const struct iovec *iov,
 				 void **desc, size_t count, fi_addr_t src_addr,
-				 void *context, uint64_t flags, uint64_t tag,
-				 uint64_t ignore)
+				 void *context, uint64_t flags, uint64_t ignore,
+				 uint64_t tag)
```

Comment:
Why did you switch the parameter order?  Seems like you went out of the way to make __ep_recvv not match __ep_recv.